build/rust/Cargo.lock

```diff
@@ -31,14 +31,15 @@ dependencies = [
 
 [[package]]
 name = "adblock"
-version = "0.3.14"
-source = "git+https://github.com/brave/adblock-rust#752712261b45de10369e43931dc30b08c9bb69fb"
+version = "0.3.15"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "5e9106f026aebe0d8924a0c520e999b406629cd5b336c0b19743542ec1092134"
 dependencies = [
  "base64 0.13.0",
  "bitflags",
  "flate2",
  "idna 0.2.3",
- "itertools 0.9.0",
+ "itertools 0.10.1",
```

Comment:
Current release is `0.10.3`. There changelog doesn't mention any security issues, so not critical, although 0.10.2 is missing.